modules/cnf-image-based-upgrade-seed-image-config.adoc

```diff
@@ -5,7 +5,7 @@
 [id="cnf-image-based-upgrade-seed-image-config_{context}"]
 = Seed image configuration
 
-The seed image targets a set of {sno} clusters with similar configuration.
+The seed image targets a set of {sno} clusters with the same hardware and similar configuration.
```

Comment:
Ref:- https://github.com/openshift-kni/lifecycle-agent/blob/main/docs/seed-image-generation.md#seed-sno-pre-requisites

I would like to update customer document with below system configuration.

`The CPU topology must align with the seed SNO. Same number of cores. Same performance tuning (i.e., reserved CPUs).`